launcher/ui/pages/modplatform/ModPage.cpp

```diff
@@ -37,10 +38,12 @@
 #include "Application.h"
 #include "ui_ModPage.h"
 
+#include <QDesktopServices>
 #include <QKeyEvent>
 #include <memory>
 
 #include <HoeDown.h>
+#include <qregularexpression.h>
```

Comment:
```suggestion
#include <QRegularExpression>
```

also put it above with the other `<Q...>` includes

Reply:
The VSCode plugin I'm using keeps doing that. Will change.